docs/user_docs/getting-started/usecase.md

```diff
@@ -110,3 +117,129 @@ Forwarding from [::1]:4000 -> 4000
 Visit [http://localhost:4000/#/github.com/KusionStack/kusion](http://localhost:4000/#/github.com/KusionStack/kusion) in your browser and enjoy.
 
 ![](/img/docs/user_docs/getting-started/gocity.png)
+
+
+## Deliver the WordPress Site with OSS as cloud storage
+
+Next, let's take a step further from the previous tutorial and set up a WordPress site on the cloud. Unlike the Code-City application, our WordPress site requires resources from the IaaS layer - Aliyun OSS - as the cloud storage for media resources, in order to save disk space for the server, reduce the workload of file migration when moving the website, and also avoid the limitation of server bandwidth when accessing files stored in independent cloud storage.
+
+### PreRequest
+
+- [Kusion](/docs/user_docs/getting-started/install)
+- [Kubernetes](https://kubernetes.io/) or [Kind](https://kind.sigs.k8s.io/)
+- This use case requires you to have an **Aliyun account** for utilizing its Object Storage Service (OSS).
+- Additionally, **a user account with AliyunOSSFullAccess permission** is required, and you can create and manage it in the [Aliyun Resource Access Management (RAM) console](https://ram.console.aliyun.com/users/). And you can configure the obtained AccessKey and SecretKey obtained as environment variables:
+
+```shell
+export ALICLOUD_ACCESS_KEY="LTAI5txxx" # replace it with your AccessKey
+export ALICLOUD_SECRET_KEY="nxuowIxxx" # replace it with your SecretKey
+```
+
+:::info
+Alternatively, Kusion provides sensitive data management tool for handling the AccessKey and SecretKey mentioned above. For detailed guidance, please refer to documentation. todo @fuyuan
```

Comment:
@howieyuen please help to complete information about kusion secret management

Reply:
yeah, kusion can fetch secret from remote, but these keys must be referenced in KCL code.
In this example, AK and SK is required for terraform CLI? not a  konfig stack

Reply:
> 
The attributes to pass the ak/sk to KCL code are already reserved: https://github.com/KusionStack/konfig/blob/main/base/pkg/kusion_clouds/alicloud_backend/alicloud_config.k#L12

Reply:
oic, thx